src/providers/cortex/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "trulens-providers-cortex" %}
+{% set version = "1.0.1a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trulens_providers_cortex-{{ version }}.tar.gz
+  sha256: 0d6be2ea0c0e578b54e90305974cf05d01480f5df2be6b1c09a80c02630e169b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9,<3.12
+    - poetry-core
+    - pip
+  run:
+    - python >=3.9,<3.12
+    - trulens-core >=1.0.0,<2.0.0
+    - trulens-feedback >=1.0.0,<2.0.0
+    - snowflake-connector-python >=3.11.0,<4.0.0
+    - snowflake-snowpark-python >=1.18.0,<2.0.0
+
+test:
+  imports:
+    - trulens.providers.cortex
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://trulens.org/
+  summary: Library to systematically track and evaluate LLM based applications.
+  license: MIT
+#  license_file: PLEASE_ADD_LICENSE_FILE
```

Comment:
To use `license_file`, we need to include an actual file in the **Source Distribution**. Let me know if you want to do it. As for now I'm going to remove this directive since it's optional